lib/utils.js

```diff
@@ -147,6 +148,16 @@ api.fetchInSecurityContext = async ({
     // since URL is for a root capability, we must dereference it
     url = url.id;
   }
+
+  // skip compaction when using specific contexts, this is a temporary measure
+  // until a zcap context is created
+  if(typeof url === 'object' && isEqual(url['@context'], [
+    constants.SECURITY_CONTEXT_V2_URL,
+    'https://w3id.org/security/suites/ed25519-2020/v1'
```

Comment:
The `isEqual` check needs to be moved into the above conditional -- as it will check for `null`/`undefined` and ensure that the root capability is always retrieved.